x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/utils.ts

```diff
@@ -162,6 +162,7 @@ export const getIndex = (
   config: LegacyServices['config']
 ): string => {
   const spaceId = getSpaceId();
-  const signalsIndex = config().get(`xpack.${APP_ID}.${SIGNALS_INDEX_KEY}`);
-  return `${signalsIndex}-${spaceId}`;
+  const signalsIndex = config().get<string>(`xpack.${APP_ID}.${SIGNALS_INDEX_KEY}`);
+
+  return spaceId ? `${signalsIndex}-${spaceId}` : signalsIndex;
```

Comment:
If they later turn on spaces I think we want this to show up with default and not be lost so to speak?

So should this be `return spaceId ? `${signalsIndex}-${spaceId}` : ${signalsIndex}-default;` so it will be active once they turn on spaces and everything they did shows up as the default?

Reply:
@FrankHassanabad I can get behind that. Hardcoding the `default` string doesn't feel great, but we can make that nicer on NP with a wrapper that handles this logic.